raiden_contracts/data/source/services/MonitoringService.sol

```diff
@@ -342,11 +344,13 @@ contract MonitoringService is Utils {
         returns (address signature_address)
     {
         bytes32 message_hash = keccak256(abi.encodePacked(
-            "\x19Ethereum Signed Message:\n148",
+            "\x19Ethereum Signed Message:\n200",
+            monitoring_service_address,
+            chain_id,
+            uint256(MessageTypeId.MSReward),
```

Comment:
If we add a message id to the signature, shouldn't we put it in the same place for all messages (e.g. first in the signature)? Otherwise there could still be conflicts (although highly unlikely). Example:

Message1 (of MessageTypeId 5):
* 10: TokenAmount
* 5: MessageTypeId
* signature

Message2 (of MessageTypeId 10):
* 10: MessageTypeId
* 5: ChannelID
* signature

Reply:
Yes, we should put the ID in the same place always.

In TokenNetwork, the message ID is always the third element after an address and a chain_id.
```
        bytes32 message_hash = keccak256(abi.encodePacked(
            signature_prefix,
            message_length,
            address(this),
            chain_id,
            uint256(MessageTypeId.Withdraw),
            channel_identifier,
            participant,
            total_withdraw
        ));
```

I can change them as well, but that causes some additional work (adjusting the client).

Reply:
I didn't expect the third place to be the fixed location. This also required the same length of the first two arguments in all cases to be safe. But I guess this is the case for us. No need to change if you watch out for this.

Reply:
I'll add the convention in the contribution guide.  I think the current convention is
```
address message_destination,
uint256 chain_id,
uin256 message_type,
...
```

I agree the third position looks like a random choice.